src/BulletDynamics/Featherstone/btMultiBodySphericalJointMotor.cpp

```diff
@@ -152,12 +156,16 @@ btQuaternion relRot = currentQuat.inverse() * desiredQuat;
 				case btMultibodyLink::eSpherical:
 				{
 					btVector3 constraintNormalAng = frameAworld.getColumn(row % 3);
-					posError = m_kp*angleDiff[row % 3];
+					double kp = m_use_multi_dof_params ? m_kp[row % 3] : m_kp[0];
+					posError = kp*angleDiff[row % 3];
+					double max_applied_impulse = m_use_multi_dof_params ? m_maxAppliedImpulseMultiDof[row % 3] : m_maxAppliedImpulse;
 					fillMultiBodyConstraint(constraintRow, data, 0, 0, constraintNormalAng,
 						btVector3(0,0,0), dummy, dummy,
-						posError,
+						posError + velocityError,
```

Comment:
adding a velocityError here is a change in behavior. Also, there is a separate argument to add a desired relative velocity. If this change is required, it is best to create a separate PR/CL for that, so we can easily undo it (if it causes issues anywhere).